### PR TITLE
Makes Regeneration and Starvation timers Settable.

### PR DESCRIPTION
**Pull request by ayyron-dev**
This Pull Request makes the Regeneration and Starvation timers Settable as seen in https://github.com/somners/Vanilla/commit/c4cb3c03d82aa0b95c68af7968e07aff54edea3b.

### CODE REVIEW

**Comment by Sleaker** [comment]
you'll need to run git rebase -i HEAD~2 to merge these commits. Leave the initial commit as p, and the followup commits as s (squash)

File: src/main/java/org/spout/vanilla/component/entity/misc/Hunger.java [hunk 1]
```diff
@@ -341,4 +343,46 @@ public void setEating(boolean eating, Slot slot) {
...
 		}
 	}
+
+	/**
+	 * Gets the start value for the Health Regeneration Start Timer length.
+	 * The Default Start Timer is 4 seconds.
+	 * @return Health Regeneration Start Timer length.
+	 */
+        public float getRegenerationTimerStart() {
+		return this.REGEN_TIMER_START;
```
Comment (kitskub) [inline src/main/java/org/spout/vanilla/component/entity/misc/Hunger.java]: Formatting...

**Comment by ayyron-dev** [comment]
Made the changes requested.

File: src/main/java/org/spout/vanilla/component/entity/misc/Hunger.java [hunk 2]
```diff
@@ -341,4 +345,68 @@ public void setEating(boolean eating, Slot slot) {
...
 		}
 	}
+
+	/**
+	 * Gets the start value for the Health Regeneration Start Timer length.
+	 * The Default Start Timer is 4 seconds.
+	 * @return Health Regeneration Start Timer length.
+	 */
+        public float getRegenerationTimerStart() {
+		return regenTimerStart > 0 ? regenTimerStart : TIMER_START;
```
Comment (kitskub) [inline src/main/java/org/spout/vanilla/component/entity/misc/Hunger.java:358]: Formatting still.

**Comment by ayyron-dev** [comment]
@kitskub that is netbeans not playing nice :/ I won't be able to fix that until tomorrow.

**Comment by Sleaker** [comment]
@somners other than formatting this looks good, thanks for taking the time, I'll check back when you've made the requested fixes!

**Comment by ayyron-dev** [comment]
@Sleaker alright, I think the formatting is all good now!

**Comment by kitskub** [comment]
Formatting is good. Can probably be pulled.

**Comment by ayyron-dev** [comment]
Alright, I'd like to add that by adding the MAX_HUNGER I am not positive that it will affect VanillaHunger.update() (https://github.com/Vanilla/Vanilla/blob/master/src/main/java/org/spout/vanilla/component/entity/player/hud/VanillaHunger.java#L141).

**Comment by kitskub** [comment]
@Sleaker why are commenting on the commit, and not the diff?

**Comment by Sleaker** [comment]
@kitskub - github happens :(

**Comment by kitskub** [comment]
@Sleaker can you review this and pull if it's ready?

**Pull request merged**